pkgs/top-level/all-packages.nix

```diff
@@ -7911,7 +7936,11 @@ in
     inherit (darwin.apple_sdk.frameworks) Security Foundation;
   };
 
-  go = go_1_12;
+  go_1_13 = callPackage ../development/compilers/go/1.13.nix {
+    inherit (darwin.apple_sdk.frameworks) Security Foundation;
+  };
+
+  go = go_1_13;
```

Comment:
It would be very nice to break this PR into two- one that introduces the go-1.13 version, and another (or several) to handle the migration of other packages.
Or if this PR is close to done, as indicated by @c00w, let's finish it up..